products/dns/terraform.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Provider::ResourceOverrides
+  ManagedZone: !ruby/object:Provider::Terraform::ResourceOverride
+    example:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dns_managed_zone_basic"
+        primary_resource_id: "prod"
+        version: <%= _version_name %>
+        vars:
+          zone_name: "prod-zone"
+    properties:
```

Comment:
can you also add an exclude for `nameServerSet`? It's not currently in TF so excluding it won't break anyone, and there's a bug in MM where it isn't the correct type so we'd have to work around that if we wanted to include it.

Reply:
Done, we can readd it in the beta provider later.